portal/portal-render-engine-impl/impl/src/webapp/vm/morpheus/moresites.vm

```diff
@@ -69,13 +69,13 @@
 
 
                     #macro( displaySite $site )
-                        <li role="presentation" class="fav-sites-entry #if (${site.isCurrentSite})is-selected #end #if (${site.isMyWorkspace})my-workspace #end">
+                        <li class="fav-sites-entry #if (${site.isCurrentSite})is-selected #end #if (${site.isMyWorkspace})my-workspace #end">
                             #if (!${site.isMyWorkspace})
-                            	<a class="site-favorite-btn" data-site-id="${site.siteId}" href="javascript:void(0);"></a>
+                                <button data-site-id="${site.siteId}" class="site-favorite-btn only-icon-btn"></button>
```

Comment:
To solve this issue, you need to...
Fix at least one (1) of these issues:

* Element does not have inner text that is visible to screen readers
* aria-label attribute does not exist or is empty
* aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
* Element has no title attribute
* Element's default semantics were not overridden with role="none" or role="presentation"

Reply:
Hi Sam, aria-label and title for this button are added through JS (line 452-472). Anyway, I've seen it could be done directly on this vm file, so I'll modify it.

Reply:
Okay, it has to remain in JS to switch between "Add / Remove [site] from favorites".

Reply:
What about changing "Add / Remove [site] from favorites" to a simple "Toggle [site] as a favorite"?

Reply:
Sounds good for the title attr, but for aria-label too? SR users may benefit from the more descriptive action.

Reply:
I agree. The more descriptive is nice. But isn't there an aria attribute that can indicate how it's currently toggled?

Reply:
Count on it.